Rule expect-expect fails for todo

In v25.2.0 (or somewhere between 25.0.5 and 25.2.0), rule `jest/expect-expect` is now failing for `it.todo`.

For example:

```js
describe('some tests', () => {
    it.todo('should have a test');
});
```

```sh
PS E:\Project>npx eslint **/*.js
E:\Project\tests\something.test.js
  2:5  warning  Test has no assertions  jest/expect-expect

✖ 1 problem (0 errors, 1 warning)
```

This occurs in Windows and v12/14/16 node containers with `eslint` 8.0.0/8.0.1.

--- a/src/rules/__tests__/expect-expect.test.ts
+++ b/src/rules/__tests__/expect-expect.test.ts
@@ -15,6 +15,8 @@ const ruleTester = new TSESLint.RuleTester({
 
 ruleTester.run('expect-expect', rule, {
   valid: [
+    'it.todo("will test something eventually")',
+    'test.todo("will test something eventually")',
     "['x']();",
     'it("should pass", () => expect(true).toBeDefined())',
     'test("should pass", () => expect(true).toBeDefined())',

--- a/src/rules/expect-expect.ts
+++ b/src/rules/expect-expect.ts
@@ -11,6 +11,7 @@ import {
   createRule,
   getNodeName,
   getTestCallExpressionsFromDeclaredVariables,
+  isSupportedAccessor,
   isTestCaseCall,
 } from './utils';
 
@@ -116,6 +117,13 @@ export default createRule<
           isTestCaseCall(node) ||
           additionalTestBlockFunctions.includes(name)
         ) {
+          if (
+            node.callee.type === AST_NODE_TYPES.MemberExpression &&
+            isSupportedAccessor(node.callee.property, 'todo')
+          ) {
+            return;
+          }
+
           unchecked.push(node);
         } else if (matchesAssertFunctionName(name, assertFunctionNames)) {
           // Return early in case of nested `it` statements.

